src/cmd-generate-hashlist

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+"""
+Creates a KeyLime Hashlist for an image.
+
+See: https://keylime.dev/
+"""
+
+import argparse
+import datetime
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+
+from cosalib import meta
+
+#: CREATE_HASH_FCCT is FCCT yaml that translate into ignition that we
+#: pass to kola to execute the hash creation script in the running system
+CREATE_HASH_FCCT="""\
+variant: fcos
+version: 1.1.0
+systemd:
+  units:
+    - name: createhash.service
+      enabled: true
+      contents: |
+          [Unit]
+          Description=Generate Keylime Hashes
+          After=network.target
```

Comment:
Hmm, why do we need this?

Reply:
We don't exactly. But I did want to ensure it executed at a sane point. `basic.target` would likely suffice.